src/common/helper.js

```diff
@@ -5,13 +5,31 @@
 const AWS = require('aws-sdk')
 const config = require('config')
 const elasticsearch = require('elasticsearch')
+const _ = require('lodash')
+const tcCoreLibAuth = require('tc-core-library-js')
+const urlencode = require('urlencode')
 
 const logger = require('./logger')
 
 AWS.config.region = config.get('esConfig.AWS_REGION')
 // ES Client mapping
 const esClients = {}
 
+const m2mAuth = tcCoreLibAuth.auth.m2m
+const util = tcCoreLibAuth.util(config)
+let m2m = null
```

Comment:
@maxceem is there any specific reason for not initializing `m2m` directly like https://github.com/topcoder-platform/tc-project-service/blob/dev/src/util.js#L28

Reply:
I don't see any particular reason which it was implemented this way. So as a result m2m would be only initialized when it first is needed, I also don't see any downsides.

Reply:
I am not sure, but I have seen something w.r.t. such initialization in past in topcoder platform it itself. However, not able to recall it as of now. If it is not too difficult, can you please convert it to the way project service is using because that is already well known way to initialize and is working fine since long time.

Reply:
Done. Now `m2m` is initialized the same way as in Project Service.

Reply:
Thanks